yarn.lock

```diff
@@ -2,36 +2,494 @@
 # yarn lockfile v1
 
 
+"@adraffy/ens-normalize@1.10.1":
```

Comment:
When did we update these dependencies? There was an npm attack earlier this week and we want to make sure we didn't get any of the affected packages.